docs/Python-Intro.rst

```diff
@@ -19,6 +19,14 @@ You can install LightGBM via pip
 ::
 
     pip install lightgbm
+    
+or conda-forge
+
+::
+
+    conda install -c conda-forge lightgbm
+    
+**Note**: The `lightgbm-feedstock <https://github.com/conda-forge/lightgbm-feedstock>`_.  is not maintained by LightGBM maintainers.
```

Comment:
```suggestion
**Note**: The `lightgbm conda-forge package <https://github.com/conda-forge/lightgbm-feedstock>`_ is not maintained by LightGBM maintainers.
```

Reply:
I'm doubt that we should document conda installation at all, because we do not maintain it and hence cannot guarantee the quality and frequency of version updates. We don't aim to document all available ways to install LightGBM via all available package managers in the world.

Reply:
I actually didn't even know about https://anaconda.org/anaconda/lightgbm that you just mentioned in https://github.com/microsoft/LightGBM/pull/3544#issuecomment-723966287 . I think that's reason enough to give some guidance in our docs about how to install with `conda`.

> We don't aim to document all available ways to install LightGBM via all available package managers in the world.

Ok but this isn't an obscure package manager like `nix` or `snap`. I think it's common, in machine learning projects in Python, to prefer `conda install` to `pip install` and I think that explicitly stating "yes this exists but we don't maintain it" in our documentation is useful.

Reply:
Sorry, I'm not 100% sold on this.

Reply:
ok, I'll defer to what you decide.

Reply:
I think we should ask other maintainers.

Reply:
ok. @henry0312 or @guolinke what do you think? The question is:

> should the LightGBM Python documentation explicitly mention installing with `conda install`, even though we don't maintain the conda packages?

Reply:
gently ping @henry0312 or @guolinke for an opinion

Reply:
sorry for missing this.
I think pip is the "lighter" way, as it requires fewer dependencies. So we prefer to install via pip.
Many machine learning tools have heavy dependencies, such as cuda, so they prefer conda, which pre-installs these dependencies.

However, I think we can still provide the conda install command, but clarify that we don't maintain it.

Reply:
thanks! Ok I agree with that. I can review from here.